python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -995,6 +1008,81 @@ def assembleSubregion(self, coaddExposure, bbox, tempExpRefList, imageScalerList
         if nImage is not None:
             nImage.assign(subNImage, bbox)
 
+    def assembleOnlineMeanCoadd(self, coaddExposure, tempExpRefList, imageScalerList, weightList,
+                                altMaskList, statsCtrl, nImage=None):
+        """Assemble the coadd using the "online" method.
+
+        This method takes a running sum of images and weights to save memory.
+        It only works for MEAN statistics.
+
+        Parameters
+        ----------
+        coaddExposure : `lsst.afw.image.Exposure`
+            The target exposure for the coadd.
+        tempExpRefList : `list`
+            List of data reference to tempExp.
+        imageScalerList : `list`
+            List of image scalers.
+        weightList : `list`
+            List of weights.
+        altMaskList : `list`
+            List of alternate masks to use rather than those stored with
+            tempExp, or None.  Each element is dict with keys = mask plane
+            name to which to add the spans.
+        statsCtrl : `lsst.afw.math.StatisticsControl`
+            Statistics control object for coadd
+        nImage : `lsst.afw.image.ImageU`, optional
+            Keeps track of exposure count for each pixel.
+        """
+        tempExpName = self.getTempExpDatasetName(self.warpType)
```

Comment:
The logging (at the debug level) that happens in `assembleSubregion` should happen here as well, and explicitly say it is using the online coaddition.